index.html

```diff
@@ -26,7 +26,16 @@
       github: "w3c/dapt",
       wgPublicList: "public-tt",
 
-      };
+      // local bibliography
+      localBiblio: {
+          "WHP051": {
```

Comment:
This reference does not seem used.

Reply:
OK, will bring in relevant description of AD that references it.